DasharoModulePkg/Library/DasharoVariablesLib/DasharoVariablesLib.c

```diff
@@ -431,7 +431,7 @@ DasharoEnableFUM (
   Status = gRT->SetVariable (
       DASHARO_VAR_FIRMWARE_UPDATE_MODE,
       &gDasharoSystemFeaturesGuid,
-      EFI_VARIABLE_BOOTSERVICE_ACCESS | EFI_VARIABLE_NON_VOLATILE,
+      EFI_VARIABLE_BOOTSERVICE_ACCESS | EFI_VARIABLE_RUNTIME_ACCESS | EFI_VARIABLE_NON_VOLATILE,

```

Comment:
Now `WarnIfFirmwareUpdateMode()` can't delete FUM variable because attributes don't match and it stays on forever.  Passing `0` for attributes in `SetVariable()` call causes variable deletion and avoids the need to synchronize attributes.

Reply:
https://github.com/Dasharo/edk2/compare/b67eb08260c40f463eca88298e548cc70f8d3831..97938d04f1df2e564c1986fd34dc0dd00467cf54#diff-306afacd6b8e1d90f20c52fe64cf2eab3b6d949cdf5225d2b310a06c84a950bcR1137 and extended description in the commit message.